app/models/team.rb

```diff
@@ -154,18 +150,12 @@ def name
     I18n.t("about.structure.#{friendly_id}.name")
   end
 
-  private def council_if_any
-    UserGroup.find_by(metadata_id: GroupsMetadataCouncils.find_by(friendly_id: self.friendly_id)&.id, metadata_type: 'GroupsMetadataCouncils')
-  end
-
   def group
-    council_if_any
+    nil
   end
```

Comment:
Not cool. Why do you need a method that has no other purpose except explicitly returning `nil`?!

Reply:
This will be replaced by the group for teams-committees.
In team_member.rb, `def role` will check if a group is available, and if it's not available, will create a group-hash.

I had two options:
1. Remove this method and remove the reference from `def role` in `team_member.rb`. Later add them back in teams-group migration (https://github.com/thewca/worldcubeassociation.org/pull/9121)
2. Make this method return nil, so that I don't have to touch `def role` in teams-group migration. In teams-group migration I'll add back the body.

I opted for 2 as the changes is slightly minimal.

Reply:
Now since the other PR is merged, this should be fine now I guess.